src/ifcopenshell-python/ifcopenshell/geom/app.py

```diff
@@ -46,13 +46,9 @@
     
 class configuration(object):
     def __init__(self):
-        try:
-            import ConfigParser
-            Cfg = ConfigParser.RawConfigParser
-        except:
-            import configparser
-            Cfg = configparser.ConfigParser(interpolation=None)
-            
+        # python2 users need to 'pip install configparser'
```

Comment:
Ugh that is a bit rough on our py2 users. While I tend to agree with @thorade to follow documented best practices, I'd like to avoid requiring all users to install yet another module for something fairly trivial as this.

I can no longer see your earlier fix. What was it? You're right about the current state that the py2 branch assigns a type object that can be instantiated and the py3 branch assigns a instance. This is clearly wrong. What about changing:

~~~diff
- Cfg = configparser.ConfigParser(interpolation=None)
+ Cfg = lambda: configparser.ConfigParser(interpolation=None)
~~~

to get sth callable into `Cfg`?